src/main/java/com/ericsson/ei/subscriptionhandler/InformSubscription.java

```diff
@@ -105,6 +105,8 @@ public void informSubscriber(String aggregatedObject, JsonNode subscriptionJson)
 		String notificationType = getSubscriptionField("notificationType", subscriptionJson);
 		String notificationMeta = getSubscriptionField("notificationMeta", subscriptionJson);
 
+		String subject = getSubscriptionField("emailSubject", subscriptionJson);
```

Comment:
As a user I would want a default subject to be added in case I did not add one. Discuss what would be best. Could be "Subscription triggered: <Subscription name>"

Reply:
Currently it is "Email Subscription Notification"... not bad